extensions/src/ACRE2TS/TsCallbacks_static.cpp

```diff
@@ -34,6 +34,8 @@ void ts3plugin_infoData(uint64 serverConnectionHandlerID, uint64 id, enum Plugin
 
     BOOL noAcre = FALSE;
     char *metaData;
+    std::string result;
```

Comment:
These should be  declared in the lowerest possible scope. In this case it is insinde the `if (ts3Functions.getClientVariableAsString(serverConnectionHandlerID, (anyID)id, CLIENT_META_DATA, &metaData) == ERROR_ok` statement